### ISSUE TITLE
[video.js] userActions property missing from VideoJsPlayerOptions

### ISSUE THREAD

**Issue by Coteh** [DefinitelyTyped/DefinitelyTyped#49821]
- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I understand that `@types` package versions [do not follow SemVer](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library).
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @joeflateau @vbortone @scleriot 

[`VideoJsPlayerOptions` is missing `userActions` property](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/01cb50d31b669834ac022f1f44bb8cf00ee09ed3/types/video.js/index.d.ts#L6872-L6900), which allows for setting hotkeys and double-click actions. See [docs](https://docs.videojs.com/tutorial-options.html#useractions) here.

**Comment by schw4rzlicht** [comment]
Stumbled upon the same issue. We will use something like the following snippet for now but it's really disturbing:
```typescript
// eslint-disable-next-line
// @ts-ignore
userActions: {
    hotkeys: true,
},
```

**Comment by joeflateau** [comment]
@Coteh @schw4rzlicht will either of you be submitting a PR for this?

**Comment by schw4rzlicht** [comment]
I will look into it :)

**Comment by joeflateau** [comment]
this is resolved by #50845
